src/basic/architecture.h

```diff
@@ -227,6 +228,10 @@ int uname_architecture(void);
 #    define native_architecture() ARCHITECTURE_ARC
 #    define LIB_ARCH_TUPLE "arc-linux"
 #  endif
+#elif defined(__loongarch64)
```

Comment:
Hmm, the order here is a mess. But please insert the new addition here right above `m68k` too.